src/exportAuthRecord.ts

```diff
@@ -122,6 +122,36 @@ async function persistAuthRecord(authRecord: Record<string, unknown>): Promise<v
     const authDir = path.join(baseAzureDir, 'ms-azuretools.vscode-azureresourcegroups');
     const authRecordPath = path.join(authDir, 'authRecord.json');
     const gitignorePath = path.join(authDir, '.gitignore');
+    // Write README.md explaining the purpose and contents of authRecord.json
+    const readmePath = path.join(authDir, 'README.md');
+    const readmeContent = [
```

Comment:
Instead of inline, can we put the readme content as a constant at the root of the file (to make indentation easier on the eyes), and use this string notation?

```typescript
const foo = `
bar
`;
```